sdk/servicebus/Azure.Messaging.ServiceBus/src/Management/SubscriptionDescription.cs

```diff
@@ -239,12 +242,14 @@ public string UserMetadata
             {
                 if (value == null)
                 {
-                    throw new ArgumentNullException(nameof(UserMetadata), $"Value cannot be null");
+                    throw new ArgumentNullException($"Value cannot be null", nameof(UserMetadata));
```

Comment:
nit: the message doesn't really add much here.
Can we use the Argument.Assert methods?